internal/server/server.go

```diff
@@ -30,11 +30,19 @@ import (
 	"cloud.google.com/go/pubsub"
 	"cloud.google.com/go/storage"
 	"github.com/datacommonsorg/mixer/internal/base"
+	pb "github.com/datacommonsorg/mixer/internal/proto"
 	"github.com/datacommonsorg/mixer/internal/store"
 	"github.com/datacommonsorg/mixer/internal/translator"
 	"github.com/datacommonsorg/mixer/internal/util"
 )
 
+// Cache holds cached data for the mixer server.
+type Cache struct {
+	ParentSvgMap   map[string]string
+	RawSvg         map[string]*pb.StatVarGroupNode
```

Comment:
Should use this in https://github.com/datacommonsorg/mixer/blob/master/internal/server/statvar_group.go#L118-L134!

Reply:
I was thinking of doing this in a followup pr

Reply:
Yep, but you may try locally see if the cache works (it should)